partials/admin-dash.html

```diff
@@ -0,0 +1,33 @@
+<section class="user-profile page">
+
+	<div class="user-profile message-wrapper">
+		<div class="user-profile header">
+			<h1>Admin Tools</h1>
+		</div>
+
+		<div class="view">
+			<!-- A GRID THAT WILL HOUSE ALL THE WIDGETS -->
+			<div class="ui grid">
+				<!-- INDIVIDUAL WIDGET -->
+				<!-- TODO REPLACE WITH ANGULAR CODE TO PROPOGATE ACROSS ALL WIDGETS -->
+				<div class="four wide column">
+					<ul>
+						<li ng-repeat="widget in widgets">
```

Comment:
I remember we thought about moving the widget over to be a little more centered. Are you still planning on doing that?